docs/src/content/docs/es/reference/frontmatter.md

```diff
@@ -258,4 +258,29 @@ title: Página para ocultar de la barra lateral autogenerada
 sidebar:
   hidden: true
 ---
+
```

Comment:
Looks like we’re missing the closing backticks for this code snippet, so the badge section is getting wrapped in a code block:
````suggestion
```

````